content/applications/sales/sales/invoicing/invoicing_policy.rst

```diff
@@ -4,78 +4,83 @@ Invoice based on delivered or ordered quantities
 
 Different business policies might require different options for invoicing:
 
-- The **Invoice what is ordered** rule is used as **default mode** in Odoo Sales, which means that
-  customers will be invoiced once the sales order is confirmed.
+- The *Invoice what is ordered* rule is used as the default mode in Odoo *Sales*, which means
+  customers are invoiced once the sales order is confirmed.
 
```

Comment:
Can remove extra line– no need for an extra line between bullet points of the same level :)
```suggestion
```